apps/docs/package.json

```diff
@@ -14,14 +14,14 @@
     "typecheck": "tsc --noEmit",
     "test": "vitest --exclude \"**/*.smoke.test.ts\"",
     "test:smoke": "npm run codegen:references && vitest -t \"prod smoke test\"",
-    "build:sitemap": "node ./internals/generate-sitemap.mjs",
+    "build:sitemap": "tsx ./internals/generate-sitemap.ts",
     "embeddings": "tsx scripts/search/generate-embeddings.ts",
     "embeddings:refresh": "npm run embeddings -- --refresh",
     "last-changed": "tsx scripts/last-changed.ts",
     "last-changed:reset": "npm run last-changed -- --reset",
     "codegen:references": "tsx features/docs/Reference.generated.script.ts",
     "codemod:frontmatter": "node ./scripts/codemod/mdx-meta.mjs && prettier --write \"content/**/*.mdx\"",
-    "postbuild": "node ./internals/generate-sitemap.mjs"
```

Comment:
It looks like `postbuild` is not run on Vercel.com https://stackoverflow.com/questions/66878927/add-post-build-script-to-next-js-on-vercel

Reply:
From the logs, it looks like it's run 🤔 

<img width="1224" alt="image" src="https://github.com/user-attachments/assets/974348b8-eae0-4d1a-afa1-1ce5e5aeb4f9">

Preview site also has the updated sitemap: https://docs-git-fix-docs-sitemap-supabase.vercel.app/docs/sitemap.xml